src/graphql/mutation/braintreeUpdateSubscriptionPaymentMethod.graphql

```diff
@@ -0,0 +1,10 @@
+mutation braintreeUpdateSubscriptionPaymentMethod($paymentMethodNonce: String!) {
+	my {
+		updateAutoDeposit (
+			autoDeposit: {
+			},
```

Comment:
what are we updating?

Reply:
I got an API error if the autoDeposit object is missing, this mutation updates the payment method, and leaves the other params intact.